tempest/api/workloadmgr/snapshot/test_snapshot_mount.py

```diff
@@ -93,3 +93,137 @@ def test_1_filesearch_default_parameters(self):
+            security_group_id = self.security_group_id
+            fvm_id = self.fvm_id
+            dismount_snapshot_id = snapshot_ids[0]
+            floating_ips_list = self.floating_ips_list
+
+            LOG.debug("dismount snasphot of a  snapshot")
+            is_dismounted = self.dismount_snapshot(dismount_snapshot_id)
+            LOG.debug("VALUE OF is_dismounted: " + str(is_dismounted))
+            if is_dismounted == True:
+                LOG.debug("dismount snapshot with full snapshot is  successful")
+                reporting.add_test_step("Verification of dismount snapshot with full snapshot", tvaultconf.PASS)
+                reporting.test_case_to_write()
+            else:
+                LOG.debug("dismount snapshot with full snapshot is unsuccessful")
+                reporting.add_test_step("Verification of dismount snapshot with full snapshot", tvaultconf.FAIL)
+                raise Exception ("Snapshot dismount with full_snapshot  does not execute correctly")
+            
+            LOG.debug("validate that snapshot is umounted from FVM")
+            self.ssh = self.SshRemoteMachineConnectionWithRSAKey(str(floating_ips_list[1]))
+            output_list = self.validate_snapshot_mount(self.ssh)
+            self.ssh.close()
+            
+            for i in output_list:
+                if '/home/ubuntu/mount_data_b' not in i:
+                   LOG.debug("connect to fvm and check mountpoint is unmounted from FVM instance")
+                   reporting.add_test_step("Verify that mountpoint is unmounted from FVM instance", tvaultconf.PASS)
+                   reporting.test_case_to_write()
+                else:
+                   LOG.debug("mount snapshot with full snapshot is unsuccessful on FVM")
+                   reporting.add_test_step("Verify that mountpoint is unmounted from FVM instance", tvaultconf.FAIL)
+                   raise Exception ("mountpoint is not showing on FVM instance")
+                if 'File_1.txt' not in i:
+                   LOG.debug("check that file is exist on mounted snapshot")
+                   reporting.add_test_step("Verification of file exist on moutned snapshot", tvaultconf.PASS)
+                   reporting.test_case_to_write()
+                else:
+                   LOG.debug("file does not found on FVM instacne")
+                   reporting.add_test_step("Verification of file exist on moutned snapshot")
+		   raise Exception ("file does not found on FVM instacne")
+        except Exception as e:
+            LOG.error("Exception: " + str(e))
+            reporting.set_test_script_status(tvaultconf.FAIL)
+            reporting.test_case_to_write()
+
+    @test.attr(type='smoke')
+    @test.idempotent_id('215e0c36-8911-4167-aaea-8c07d21212f3')
+    def test_3_snapshot_mount_incremental(self):
+        reporting.add_test_script(str(__name__) + "_incremental_snasphot")
+        try:
+            if self.exception != "":
```

Comment:
As same above this test case does not follow any prerequisite call, this condition can be removed.